framework/include/kernels/CoefReaction.h

```diff
@@ -21,11 +21,22 @@ class CoefReactionTempl : public ReactionTempl<is_ad>
 
 protected:
   virtual GenericReal<is_ad> computeQpResidual() override;
-  virtual Real computeQpJacobian() override;
 
   /// input parameter multiplied by the reaction kernel
   const Real _coef;
 };
 
-typedef CoefReactionTempl<false> CoefReaction;
+class CoefReaction : public CoefReactionTempl<false>
+{
+public:
+  static InputParameters validParams();
+
+  CoefReaction(const InputParameters & parameters);
+
+  using CoefReactionTempl<false>::CoefReactionTempl;
+
+protected:
+  virtual Real computeQpJacobian() override;
+};
```

Comment:
what's the reason for these new lines? It seems like things were already working before with the simple typedefs ?

Reply:
It's due to needing to override `computeQpJacobian` due to changes in `Reaction`. I couldn't figure out a better way to do it.